cubes/new_cube.py

```diff
@@ -189,8 +199,6 @@ def main(Cube):
         if not os.path.exists(outdir):
             os.mkdir(outdir)
         
-        #psetmins,psetmaxes,nvals=misc_functions.process_pfile(Cube[2])
-        vparam_dict=misc_functions.process_jfile(Cube[2])
         run=Cube[0]
```

Comment:
Can you add some comments on whether this is cubing over multiple dimensions or just a single vector by default? i.e. is this syntax general or for only 1D?

Reply:
This follows exactly what we had before,
which I'll admit I don't fully follow.  So, 
I'm not sure what to comment here. ;)